.vscode/launch.json

```diff
@@ -44,224 +44,60 @@
             "preLaunchTask": "buildDev"
         },
         {
-            "type": "node",
-            "request": "launch",
-            "name": "Omnisharp: Launch Unit tests",
-            "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
-            "args": [
-                "-u",
-                "tdd",
-                "--timeout",
-                "999999",
-                "--colors",
-                "${workspaceFolder}/out/omnisharptest/omnisharpUnitTests/**/*.test.js"
-            ],
-            "sourceMaps": true,
-            "internalConsoleOptions": "openOnSessionStart",
-            "outFiles": [
-                "${workspaceRoot}/out/omnisharptest/**/*.js"
-            ],
-            "preLaunchTask": "build"
-        },
-        {
-            "name": "Omnisharp: Launch Feature Tests",
+            "name": "Omnisharp: Launch Current File Integration Tests",
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
             "args": [
                 // Create a temp profile that has no extensions / user settings.
                 // This allows us to only have the C# extension + the dotnet runtime installer extension dependency.
                 "--profile-temp",
+                "${workspaceRoot}/omnisharptest/omnisharpIntegrationTests/testAssets/${input:omnisharpAssetName}/.vscode/omnisharp_${input:omnisharpAssetName}.code-workspace",
```

Comment:
removed all the configs for the various asset files.  Now when this is selected, we show a quick pick to choose which assets to use.

Reply:
![image](https://github.com/dotnet/vscode-csharp/assets/5749229/a2abb774-f7ad-4e4f-825e-1c7831f1a00a)